.github/workflows/antithesis-test.yml

```diff
@@ -75,8 +75,8 @@ jobs:
       - name: Build and push client image
```

Comment:
Don't know if we should do it now, but I would prefer we consolidate code executed by workflows and contributors and just call `make antithesis-build-client-docker-image` instead of trying to keep two separate docker configurations.